CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 
 - Terminal escape bindings with combined modifiers for Delete and Insert
+- Added /Applications symlink into OS X DMG for easier installation
```

Comment:
```suggestion
- Symlink to /Applications in macOS DMG for easier installation
```

We usually try not to prepend `Added` whenever possible, otherwise the entire `### Added` section would look like `- Added ...` which is just unnecessary noise.